src/sonic-yang-models/sonic-acl.yang

```diff
@@ -0,0 +1,231 @@
+module sonic-acl {
+	namespace "http://github.com/Azure/sonic-acl";
+	prefix sacl;
+	yang-version 1.1;
+
+	import ietf-inet-types {
+		prefix inet;
+	}
+
+	import sonic-common {
+		prefix cmn;
+	}
+
+	import sonic-extension {
+		prefix sonic-ext;
+	}
+
+	import sonic-port {
+		prefix prt;
+	}
+
+	import sonic-mirror-session {
+		prefix sms;
+	}
+
+	organization
+		"SONiC";
+
+	contact
+		"SONiC";
+
+	description
+		"SONIC ACL";
+
+	revision 2019-05-15 {
+		description
+			"Initial revision.";
+	}
+
+	container sonic-acl {
+
+		container ACL_TABLE {
+
+			list ACL_TABLE_LIST {
+				key "aclname";
+				max-elements 1024; // Max 1K ACL tables for all platforms
+				sonic-ext:custom-validation ValidateMaxAclTable;
+
+				leaf aclname {
+					type string {
+						pattern '[a-zA-Z0-9]{1}([-a-zA-Z0-9_]{0,71})';
+						length 1..72;
+					}
+				}
+
+				leaf policy_desc {
+					type string {
+						length 1..255 {
+							error-app-tag policy-desc-invalid-length;
+						}
+					}
+				}
+
+				leaf stage {
+					type enumeration {
+						enum INGRESS;
+						enum EGRESS;
+					}
+				}
+
+				leaf type {
+					type enumeration {
+						enum MIRROR;
+						enum L2;
+						enum L3;
+						enum L3V6;
+					}
+				}
+
+				leaf-list ports {
+					type leafref {
```

Comment:
Port Channel also can be added in ACL-TABLE

Reply:
Sure you can extend that.